python/ray/tests/test_basic.py

```diff
@@ -850,6 +850,7 @@ def g():
                 args=[], num_cpus=1, num_gpus=1,
                 resources={"Custom": 1})) == [0]
         infeasible_id = g._remote(args=[], resources={"NonexistentCustom": 1})
+        assert ray.get(g._remote()) == [0]
```

Comment:
This change looks good. However, it looks like there is a pre-existing problem with the entire test, which is that it is indented too far. The whole `def test_submit_api(shutdown_only):` test needs to be unindented by four spaces. As it currently is, it isn't getting run.

Note that the test is good, but it's actually going to fail when you run it. The reason is that you need to modify `_remote` in `python/ray/remote_function.py` to set `args = []` and `kwargs = {}` if they are not provided.